crypto/cryptlib.c

```diff
@@ -272,6 +281,21 @@ void OPENSSL_showfatal(const char *fmta, ...)
     va_end(ap);
 
 # if defined(_WIN32_WINNT) && _WIN32_WINNT>=0x0333
+#  ifdef OPENSSL_SYS_WIN_CORE
+    /* ONECORE is always NONGUI and NT >= 0x0601 */
+
+    /* TODO: For non GUI and no std error cases - Add event logging feature here. */
```

Comment:
Coding style guide defines maximum line length at 80. This line is longer. Also note that multi-line comments are supposed to have opening `/*` and closing `*/` on separate lines.

And while you are at it, it would more as appropriate to apply same length limitation to commentary in 50-win-onecore.conf.